src/common/datadog.config.js

```diff
@@ -0,0 +1,18 @@
+import { datadogRum } from '@datadog/browser-rum';
+
+datadogRum.init({
+    applicationId: '3937053e-386b-4b5b-ab4a-c83217d2f953',
```

Comment:
Will this be the same for all environments?

Reply:
I think so